src/app/navigation/application-launcher.component.ts

```diff
@@ -0,0 +1,79 @@
+import {
+    Component, ElementRef, EventEmitter,
+    Input, OnInit, Output, Renderer2,
+    TemplateRef, ViewEncapsulation
+} from '@angular/core';
+
+import { NavigationItemConfig } from './navigation-item-config';
+import { read } from 'fs-extra';
+
+@Component({
+    encapsulation: ViewEncapsulation.None,
+    selector: 'pfng-application-launcher',
+    templateUrl: './application-launcher.component.html'
+})
+
+export class ApplicationLauncherComponent  implements OnInit {
```

Comment:
The ApplicationLauncherComponent class needs some doc. Each class and public method shows up in the TS documentation.